diff --git a/edk2toolext/environment/extdeptypes/nuget_dependency.py b/edk2toolext/environment/extdeptypes/nuget_dependency.py
index c3d95ee..679fd99 100644
--- a/edk2toolext/environment/extdeptypes/nuget_dependency.py
+++ b/edk2toolext/environment/extdeptypes/nuget_dependency.py
@@ -59,17 +59,17 @@ def GetNugetCmd(cls: "NugetDependency") -> list[str]:
             (None): none was found
         """
         cmd = []
-        if GetHostInfo().os != "Windows":
-            cmd += ["mono"]
+        # if GetHostInfo().os != "Windows":
+        #     cmd += ["mono"]
 
         nuget_path = os.getenv(cls.NUGET_ENV_VAR_NAME)
-        if nuget_path is not None:
-            nuget_path = os.path.join(nuget_path, "NuGet.exe")
-            if not os.path.isfile(nuget_path):
-                logging.info(f"{cls.NUGET_ENV_VAR_NAME} set, but did not exist. Attempting to download.")
-                DownloadNuget(nuget_path)
-        else:
-            nuget_path = DownloadNuget()
+        # if nuget_path is not None:
+        #     nuget_path = os.path.join(nuget_path, "NuGet.exe")
+        #     if not os.path.isfile(nuget_path):
+        #         logging.info(f"{cls.NUGET_ENV_VAR_NAME} set, but did not exist. Attempting to download.")
+        #         DownloadNuget(nuget_path)
+        # else:
+        #     nuget_path = DownloadNuget()
 
         if not os.path.isfile(nuget_path):
             logging.error("We weren't able to find or download NuGet!")
