diff --git a/source/Makefile b/source/Makefile
index 602ddf0..ba93b58 100644
--- a/source/Makefile
+++ b/source/Makefile
@@ -48,6 +48,9 @@ ifeq ($(MAKECMDGOALS), modules)
 		CC=$(CROSS_COMPILE)gcc LD=$(CROSS_COMPILE)ld \
 		NV_KERNEL_SOURCES=$(KERNEL_HEADERS) \
 		NV_KERNEL_OUTPUT=$(KERNEL_OUTPUT) \
+		KBUILD_EXTMOD=$(NVIDIA_CONFTEST)/nvidia \
+		srctree.nvidia=$(MAKEFILE_DIR)/nvidia-oot \
+		IGNORE_MISSING_MODULE_SYMVERS=1 \
 		-f $(NVIDIA_CONFTEST)/nvidia/Makefile
 endif
 
@@ -67,6 +70,7 @@ hwpm: conftest
 		CONFIG_TEGRA_OOT_MODULE=m \
 		srctree.hwpm=$(MAKEFILE_DIR)/hwpm \
 		srctree.nvconftest=$(NVIDIA_CONFTEST) \
+		KCFLAGS="-I$(NVIDIA_CONFTEST) -I$(NVIDIA_CONFTEST)/nvidia" \
 		$(MAKECMDGOALS)
 
 nvidia-oot: conftest hwpm