diff --git a/nvdisplay/kernel-open/Kbuild b/nvdisplay/kernel-open/Kbuild
index 2470c70..f36bca3 100644
--- a/nvdisplay/kernel-open/Kbuild
+++ b/nvdisplay/kernel-open/Kbuild
@@ -73,6 +73,8 @@ EXTRA_CFLAGS += -I$(src)
 EXTRA_CFLAGS += -Wall $(DEFINES) $(INCLUDES) -Wno-cast-qual -Wno-error -Wno-format-extra-args
 EXTRA_CFLAGS += -D__KERNEL__ -DMODULE -DNVRM
 EXTRA_CFLAGS += -DNV_VERSION_STRING=\"540.3.0\"
+EXTRA_CFLAGS += -Wno-incompatible-pointer-types
+EXTRA_CFLAGS += -Wno-int-conversion
 
 ifneq ($(SYSSRCHOST1X),)
  EXTRA_CFLAGS += -I$(SYSSRCHOST1X)
diff --git a/nvgpu/drivers/gpu/nvgpu/Makefile b/nvgpu/drivers/gpu/nvgpu/Makefile
index 72e0e95..b893738 100644
--- a/nvgpu/drivers/gpu/nvgpu/Makefile
+++ b/nvgpu/drivers/gpu/nvgpu/Makefile
@@ -23,6 +23,7 @@ ccflags-y += -Wno-multichar
 ccflags-y += -Werror
 ccflags-y += -Wno-error=cpp
 ccflags-y += -Wmissing-prototypes
+ccflags-y += -Wno-implicit-function-declaration
 ifeq ($(VERSION),4)
 ccflags-y += -Wextra -Wno-unused-parameter -Wno-missing-field-initializers
 endif
