diff --git a/nvidia-oot/scripts/conftest/conftest.sh b/nvidia-oot/scripts/conftest/conftest.sh
index a7fb019..9453c68 100755
--- a/nvidia-oot/scripts/conftest/conftest.sh
+++ b/nvidia-oot/scripts/conftest/conftest.sh
@@ -7250,7 +7250,7 @@ compile_test() {
             #include <linux/compiler_attributes.h>
             #include <linux/shrinker.h>
             int conftest_register_shrinker_has_fmt_arg(struct shrinker *s, const char *name) {
-                    return register_shrinker(s, name);
+                    return register_shrinker(s, \"%s\", name);
             }"
 
             compile_check_conftest "$CODE" "NV_REGISTER_SHRINKER_HAS_FMT_ARG" "" "types"

